packages/api-server/tests/utils/config.test.ts

```diff
@@ -0,0 +1,209 @@
+import { HexString, Script } from "@ckb-lumos/base";
+import {
+  BackendType,
+  EoaScriptType,
+  GodwokenClient,
+  GwScriptType,
+  NodeInfo,
+  NodeMode,
+} from "@godwoken-web3/godwoken";
+import test from "ava";
+import { GwConfig } from "../../src/base/gw-config";
+
+const script: Script = {
+  code_hash: "0x",
+  hash_type: "type",
+  args: "0x",
+};
+
+const gwConfig = new GwConfig("http://host:8119");
+
+let mockRpc: GodwokenClient = gwConfig.rpc;
```

Comment:
I think a related Godwoken-Kicker PR is required to test this feature.

Reply:
pls note this is only for unit test. the integration test should be added after we have setup pr #263